soccer/planning/RRTPlanner.cpp

```diff
@@ -195,109 +212,349 @@ float getTime(InterpolatedPath& path, int index,
         motionConstraints.maxAcceleration, startSpeed, endSpeed);
 }
 
-std::unique_ptr<Planning::InterpolatedPath> RRTPlanner::generateVelocityPath(
-    std::vector<Geometry2d::Point>& points,
+std::unique_ptr<InterpolatedPath> RRTPlanner::generatePath(
```

Comment:
Yeah it makes sense that this could be applied to paths generated in other ways.  I'm just asking about this method in particular.  If all it's doing is calling `generateCubicBezier()` by forwarding the parameters exactly as is, why not just cut it out?  I think the name `generateCubicBezier()` is more descriptive anyways.